empress/core.py

```diff
@@ -164,7 +164,10 @@ def __init__(self, tree, table=None, sample_metadata=None,
 
                 # if there are no matches set to None so Emperor can ignore
                 # the feature metadata
-                feature_metadata = pd.concat([self.tip_md, self.int_md])
+                if (self.tip_md is None) and (self.int_md is None): 
```

Comment:
Parenthesis are not needed here, and the PEP-8 linter will error:

```suggestion
                if self.tip_md is None and self.int_md is None: 
```